apps/fabric-website/src/components/Nav/Nav.tsx

```diff
@@ -187,40 +187,38 @@ export class Nav extends React.Component<INavProps, INavState> {
     );
   }
 
-  private _getSearchBox(val, page) {
-    if (val === 'Components' && _isPageActive(page)) {
-      return (
-        <div className={css(styles.searchBox)}>
-          <SearchBox
-            placeholder="Filter Components"
-            underlined={true}
-            styles={searchBoxStyles}
-            onChange={this._onChangeQuery.bind(this)}
-          />
-          <IconButton
-            iconProps={{ iconName: 'filter' }}
-            style={{ color: 'white', marginLeft: '5px' }}
-            menuIconProps={{ iconName: '' }}
-            menuProps={{
-              items: [
-                {
-                  key: 'categories',
-                  text: 'Categories',
-                  iconProps: { iconName: 'org' },
-                  onClick: this._setCategories.bind(this)
-                },
-                {
-                  key: 'alphabetized',
-                  text: 'A to Z',
-                  iconProps: { iconName: 'Ascending' },
-                  onClick: this._setAlphabetized.bind(this)
-                }
-              ]
-            }}
-          />
-        </div>
-      );
-    }
+  private _getSearchBox() {
+    return (
+      <div className={css(styles.searchBox)}>
```

Comment:
what's the point of wrapping in `css` here? there is a single string classname...